src/applications/personalization/dashboard-2/components/claims-and-appeals/Appeal.jsx

```diff
@@ -101,7 +101,7 @@ const Appeal = ({ appeal, name }) => {
         </div>
       </div>
       <CTALink
-        aria-label={`View details of ${appealTitle} `}
+        ariaLabel={`View details of ${appealTitle} `}
```

Comment:
OMG, RTL FTW. Four existing tests, [like this one](https://github.com/department-of-veterans-affairs/vets-website/blob/bef1988043b3f5adb82522be93d0310ab70cd342/src/applications/personalization/dashboard-2/components/claims-and-appeals/ClaimsAndAppeals.unit.spec.jsx#L363-L366), failed because I was using the wrong prop name.